serving/docs/lmi/deployment_guide/configurations.md

```diff
@@ -125,18 +125,17 @@ You can find these configurations in the respective [user guides](../user_guides
 
 All LMI Configuration keys available in the `serving.properties` format can be specified as environment variables.
 
-The translation for `engine` is unique. The configuration `engine=<engine>` is translated to `SERVING_LOAD_MODELS=test::<engine>=/opt/ml/model`.
-For example:
-
-* `engine=Python` is translated to environment variable `SERVING_LOAD_MODELS=test::Python=/opt/ml/model`
```

Comment:
By default, LMI will use `engine=Python` will be used, we can use `OPTION_ENGINE=Python` to explictly set the engine.

You can also force use `OPTION_MPI_MODE=true` to force use MPI mode